CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to LinuxBoot/Heads
+
+Thank you for your interest in contributing to LinuxBoot/Heads! We welcome contributions
+from everyone. By participating in this project, respecting these contribution guidelines
+will ease and fasten their review, fixing and merging.
+
+## Communication Channels
+
+For the most up-to-date information on our community channels, please visit:
+https://osresearch.net/community/
+
+## Best Practices for Issues and Contributions
+
+Before opening an issue or submitting a contribution, please:
+
+1. Search existing resources (GitHub issues, wiki, Matrix channel history).
+2. Review the documentation at https://osresearch.net.
+3. Choose the appropriate channel for your contribution or question.
+
+### When Starting a Discussion on Matrix
+
+- Clearly state your topic or question.
+- Provide context, including your environment (OS, hardware, etc.) if relevant.
+- Be patient and respectful while waiting for responses.
```

Comment:
Some things might effectively get lost in a stream of notifications, so I'd mention pinging.
```suggestion
- Be patient and respectful while waiting for responses. Feel free to "ping" (draw attention) after some time if it looks like your question wasn't noticed.
```